.pylintrc

```diff
@@ -108,6 +108,7 @@ disable=
     C0302, # too-many-lines
     W9013, # missing-yield-doc
     W9014, # missing-yield-type-doc
+    R0801, # duplicate-code, cannot be individually disabled (https://github.com/PyCQA/pylint/issues/214), enable when pylint updated
```

Comment:
we are already using the latest pylint and the issue seems to be closed

Reply:
Aaaa good call, I tried a number of ways to put the disable comment but didn't work, finally figured out it has to be under the `class`...

Reply:
😮‍💨